src/makeCalculator.js

```diff
@@ -3,8 +3,44 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(operate) {
+  const obj = {
```

Comment:
```suggestion
  return {
```

no need to create variable here - you can immediately return it
BTW, obj is bad name 🙂